lib/core/unit-tests/test_ink_inet.cc

```diff
@@ -21,14 +21,14 @@
     limitations under the License.
 */
 
-#include <ts/TextView.h>
-#include <ts/BufferWriter.h>
-#include <ts/ink_inet.h>
+#include <tsutil/TextView.h>
```

Comment:
Why have tsutil and core?  We should have only one base library.  Also, I don't like the name core since it is confusing with iocore.

Reply:
Because you requested that on the mailing list. You were concerned about C++ ABI compatibility checks being difficult if there was a single library for code used only in the core and code also used externally by plugin, and recommended they be split out.

I'm open to other suggestions for "core". I don't think it's much of a big deal with regard to "iocore" because that's going to go away in the not distant future as things move to "src".

Reply:
iocore components should be moved to lib/.  I am ok with core since iocore will be going away.